src/main/java/array/Ex5_Tongsonguyento.java

```diff
@@ -0,0 +1,32 @@
+package array;
+
+public class Ex5_Tongsonguyento {
+    public static boolean nguyento(int n) {
+        if (n < 2) {
+            return false;
+        } else {
+            if (n > 2) {
+                if (n % 2 == 0) {
+                    return false;
+                } else {
+                    for (int i = 3; i < (float) Math.sqrt(n); i++) {
+                        if (n % i == 0) {
+                            return false;
+                        }
+                    }
+                }
+            }
+            return true;
+        }
+    }
+
+    public int tong_so_nto(int []a) {
+        int sum = 0;
+        for (int i = 0; i < a.length; i++) {
```

Comment:
Có thể dùng [foreach](https://www.geeksforgeeks.org/for-each-loop-in-java/)  `for(int number : a)`

Reply:
done